.github/workflows/image.yaml

```diff
@@ -0,0 +1,48 @@
+# (C) Copyright Red Hat 2022.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build and push container images for each cloud provider.
+---
+name: image
+on:
+  push:
+    branches:
+      - 'staging'
+jobs:
+  build_push_job:
+    name: build and push
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        provider:
+          # Please keep this list in alphabetical order.
+          - aws
+          - azure
+          - ibmcloud
+          - libvirt
```

Comment:
Should we also build the vsphere image here?

Reply:
I am not sure regarding it because I don't have the context. I checked the [images](https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/.github/workflows/build.yaml#L15) we were testing builds for and copied the list or the same. If it makes sense, I guess we should add it. @bpradipt can give more context.

Reply:
Ok, no worries. I just looked on quay and saw there was an image that Pradipta built yesterday for it:
<img width="485" alt="image" src="https://user-images.githubusercontent.com/35110159/196382286-02201716-08ad-4990-8834-0d10d46049da.png">
, so mentioned it, but lets wait for Pradipta assessment before making changes. Thanks!

Reply:
Yeah, let's build the image for vsphere as well.

Reply:
Done